pkg/util/log/logtestutils/log_spy.go

```diff
@@ -79,6 +79,13 @@ func (s *LogSpy) ReadAll() []logpb.Entry {
 	return s.readLocked(math.MaxUint32)
 }
 
+// Clear clears the logs contained within the spy.
+func (s *LogSpy) Clear() {
```

Comment:
nit: The `StructuredLogSpy` has this functionality named as `Reset`, can we rename this to be the same?